### PR TITLE
[ADD] l10n_pe: add new taxes for localization

### PR DESCRIPTION
**Pull request by umiphos**
Description of the issue/feature this PR addresses:

Add new taxes to avoid configuration of this taxes when instaling a new database. Both taxes are already tested.

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Review by nhomar — APPROVED** [review]
@jco-odoo

**Comment by umiphos** [comment]
Taking the demo company from  https://github.com/odoo/odoo/pull/53107

File: addons/l10n_pe/data/account_fiscal_position_template_data.xml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
```
Comment (jco-odoo) [inline addons/l10n_pe/data/account_fiscal_position_template_data.xml:2]: @nhomar Best to have an empty fiscal position with country_id Peru auto_apply True before this one.  And you can put this one in auto_apply as well.

File: addons/l10n_pe/data/account_tax_data.xml [hunk 2]
```diff
@@ -224,6 +228,68 @@
...
+        <field name="include_base_amount">1</field>
+        <field name="tax_group_id" ref="tax_group_ivap"/>
+    </record>
+    <record id="sale_tax_icbper" model="account.tax.template">
+        <field name="chart_template_id" ref="pe_chart_template"/>
+        <field name="name">.2% ICBPER</field>
+        <field name="description">ICBPER</field>
+        <field name="l10n_pe_edi_tax_code">7152</field>
+        <field name="l10n_pe_edi_unece_category">S</field>
+        <field name="amount">.2</field>
```
Comment (jco-odoo) [inline addons/l10n_pe/data/account_tax_data.xml:286]: What is against 0.2?

File: addons/l10n_pe/demo/l10n_pe_demo.xml [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+        <field name="phone">+51 912 345 678</field>
+        <field name="email">info@company.peexample.com</field>
+        <field name="website">www.peexample.com</field>
+    </record>
+    <record id="demo_company_pe" model="res.company">
+        <field name="name">PE Company</field>
+        <field name="partner_id" ref="partner_demo_company_pe"/>
+    </record>
+
+    <record id="res_partner_1" model="res.partner">
```
Comment (jco-odoo) [inline addons/l10n_pe/demo/l10n_pe_demo.xml:20]: I suppose l10n_pe.res_partner_1 will be different than the base one then.  But another xml_id might be more logical.

File: addons/l10n_pe/data/account_tax_data.xml [hunk 4]
```diff
@@ -224,6 +228,68 @@
...
+        <field name="amount_type">fixed</field>
+        <field name="type_tax_use">sale</field>
+        <field name="sequence">1</field>
+        <field name="include_base_amount">1</field>
```
Comment (jco-odoo) [inline addons/l10n_pe/data/account_tax_data.xml:290]: Are you sure?

**Review by jco-odoo — CHANGES_REQUESTED** [review]
Some small comments, but please check.

File: addons/l10n_pe/data/account_tax_data.xml (hunk 2 above)
Comment (umiphos) [reply addons/l10n_pe/data/account_tax_data.xml:286]: hi, this is a fixed tax it will apply per unit, this is the amount for 2020

Comment (jco-odoo) [reply addons/l10n_pe/data/account_tax_data.xml:286]: Nice, but it does not answer the question.  I feel like in the 80s again with this notation.

**Comment by jco-odoo** [comment]
@umiphos ok?

File: addons/l10n_pe/data/account_tax_data.xml [hunk 5]
```diff
@@ -224,6 +228,67 @@
...
+        <field name="l10n_pe_edi_unece_category">S</field>
+        <field name="amount">4</field>
+        <field name="type_tax_use">sale</field>
+        <field name="sequence">1</field>
+        <field name="include_base_amount">1</field>
+        <field name="tax_group_id" ref="tax_group_ivap"/>
+    </record>
+    <record id="sale_tax_icbper" model="account.tax.template">
+        <field name="chart_template_id" ref="pe_chart_template"/>
+        <field name="name">0.2% ICBPER</field>
```
Comment (jco-odoo) [inline addons/l10n_pe/data/account_tax_data.xml:282]: @umiphos Why is this percentage and the amount is fixed?

Comment (jco-odoo) [reply addons/l10n_pe/data/account_tax_data.xml:282]: Another way to handle this is to put a type price included 100%, such that users just create a plastic bag product with a price of 0.2 as the product is taxed entirely.  @umiphos @nhomar

File: addons/l10n_pe/data/account_fiscal_position_template_data.xml (hunk 1 above)
Comment (nhomar) [reply addons/l10n_pe/data/account_fiscal_position_template_data.xml:2]: done

File: addons/l10n_pe/data/account_tax_data.xml (hunk 5 above)
Comment (nhomar) [reply addons/l10n_pe/data/account_tax_data.xml:282]: We will do it on this way that you mentioned.

Comment (nhomar) [reply addons/l10n_pe/data/account_tax_data.xml:282]: @jco-odoo 

Sorry I think incomplete my previous answer.

The tax is over any plastic bag, then if I sell a plastic bag on 2 the price will be 2*1.18(VAT)+0.2==2.56 then create a product do not make much sense.

FYI: the change we did with your proposal I did not test it with multiple taxes per line and it is broken, I will revert it temporarily to show you on the next meeting the effect.

Comment (nhomar) [reply addons/l10n_pe/data/account_tax_data.xml:282]: > @umiphos Why is this percentage and the amount is fixed?

We will fix that it is a typo!

**Comment by jco-odoo** [comment]
@umiphos I suppose we have all this in v14 already?  Of course we can still merge this if necessary.

**Comment by jco-odoo** [comment]
@nhomar No idea why this was still open

**Pull request closed without merging**